packages/wonder-blocks-tooltip/components/tooltip-bubble.js

```diff
@@ -11,7 +11,7 @@ import TooltipTail from "./tooltip-tail.js";
 
 import type {getRefFn, Offset, Placement} from "../util/types.js";
 
-export type TooltipBubbleProps = {|
```

Comment:
Changed the name here to make it more generic (now that it's also being used by Popover). What do you think @somewhatabstract?

Reply:
I'm still uncomfortable making some of these things public API elements and the increased support surface area that implies. But as far as naming goes, it seems reasonable to me.